src/Symfony/Component/Yaml/Inline.php

```diff
@@ -89,12 +92,13 @@ public static function parse($value, $exceptionOnInvalidType = false, $objectSup
      * @param mixed $value                  The PHP variable to convert
      * @param bool  $exceptionOnInvalidType true if an exception must be thrown on invalid types (a PHP resource or object), false otherwise
      * @param bool  $objectSupport          true if object support is enabled, false otherwise
+     * @param bool  $timestampAsDateTime    true if DateTime objects must be dumped as YAML timestamps, false if DateTime objects are not supported
```

Comment:
you don't need this ``$timestampAsDateTime`` argument when dumping, given that you don't respect it anyway